projects/ViTDet/vitdet/custom_layer_norm.py

```diff
@@ -0,0 +1,17 @@
+from mmengine.registry import MODELS
+from torch import Tensor
+from torch.nn import LayerNorm
+
+
+@MODELS.register_module()
+class CustomLN(LayerNorm):
+    """Custom Layer Normalization."""
+
+    def forward(self, x: Tensor) -> Tensor:
+        if x.dim() == 4:
+            x = x.permute(0, 2, 3, 1)
+            x = super().forward(x)
+            x = x.permute(0, 3, 1, 2).contiguous()
+        else:
+            x = super().forward(x)
+        return x
```

Comment:
How about using this https://github.com/open-mmlab/mmclassification/blob/3d4f80d63c6ac929d487a512ee2cf0cd53c47ed8/mmcls/models/backbones/convnext.py#L19-L20 if this project supposes `MMClassificatoin` is already installed.

Reply:
We also need to support x.dim() == 3 in this PR.

Reply:
I understand. Thank you!

Reply:
Good idea, we will take your advice.